blint/cli.py

```diff
@@ -86,6 +86,20 @@ def build_parser():
         dest="suggest_fuzzable",
         help="Suggest functions and symbols for fuzzing based on a dictionary.",
     )
+    # TODO: what to do should this be default?
+    parser.add_argument(
+        "--use-blintdb",
+        action="store_true",
+        default=True,
+        dest="use_blintdb",
+        help="Use blintdb for symbol resolution.",
+    )
+    # TODO: Please suggest if this location is good
+    parser.add_argument(
```

Comment:
@aryan-rajoria Let's handle this [like we do VDB_HOME](https://github.com/AppThreat/vulnerability-db/blob/147392be18ae4a3b63ea34d509cc2a2dc8765381/vdb/lib/config.py#L23) - rather than using a command line argument, check for an environment variable. If not set, use the appdirs user_data_dir function to set the directory.